### PR TITLE
Further improve Plugwise documentation

### PR DESCRIPTION
**Pull request by CoMPaTech**
## Proposed change
Overall improvements to the Plugwise documentation using the quality_scale as a guideline and improve readability for (and from) end-user perspective.

Starting with a better introduction and overview, before diving into platforms (which are now seperate sections). Moving troubleshooting items and removal more towards the final sections. Adding (some) tables for improved reading on data-updates and platforms as well as showing the action examples per platform. Improved troubleshooting/diagnostic instructions.

Overview as per requested changes from the initial benchmark by @joostlek

- [x] `docs-high-level-description` via https://github.com/home-assistant/core/pull/131888#discussion_r1865806781 - Improved introduction tekst and restructured/reordered accordingly for improved approach (by aligning with comparable integrations)
- [x] multiple docs related via https://github.com/home-assistant/core/pull/131888#discussion_r1865818063
  - [x] `docs-use-cases` - added through various sections and examples (and included gas explicitely)
  - [x] `docs-supported-devices` - reworked this section for improved reading including the top section to exclude formerly sold USB (directly)
  - [x] `docs-supported-functions` - reworked by platform section for clear structure
  - [x] `docs-data-update` - was already adjusted in earlier version, though reworked to include gas
  - [x] `docs-known-limitations` - added as a section, ingesting existing remarks and added new
  - [x] `docs-troubleshooting` - added as a section, ingesting existing
  - [x] `docs-examples` - the existing sample actions have been assigned to their respective platform sections, generally added more use-case directives

This continues on from #36087 effort on:

- [x] `docs-installation-instructions` via https://github.com/home-assistant/core/pull/131888#discussion_r1865811660 - implemented in docs PR 36087
- [x] `docs-removal-instructions` via https://github.com/home-assistant/core/pull/131888#discussion_r1865811759 - implemented in docs PR 36087
- [x] `docs-installation-parameters` via https://github.com/home-assistant/core/pull/131888#discussion_r1865813463 - implemented in docs PR 36087

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository:
- This PR fixes or closes issue: fixes https://github.com/home-assistant/core/pull/131888

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Enhanced clarity and detail in the Plugwise integration documentation.
	- Restructured sections for supported devices and platforms.
	- Clarified prerequisites for integration usage, including the need for Gateway ID as the password.
	- Reformatted data update intervals into a table for better readability.
	- Expanded actions and examples with clearer YAML script examples for functionalities.
	- Improved troubleshooting section with detailed steps for accessing local devices and modifying update intervals.
	- Specified lack of support for legacy power devices using a USB stick directly.

### CODE REVIEW

**Comment by CoMPaTech** [comment]
@joostlek updated all 'unresolved's related to this PR in https://github.com/home-assistant/core/pull/131888 (as they are linked from this PR description

File: source/_integrations/plugwise.markdown [hunk 1]
```diff
@@ -30,19 +30,30 @@ ha_platforms:
...
+- Although no longer sold, there also is support for Stretch, a gateway to create network connectivity for their older power products.
+
+Plugwise formerly sold power based products using a USB stick as the controller. This integration does not support the `Stick` directly, see [legacy power devices](#legacy-power-devices) for more information.
+
+## Platforms
+
+Depending on your specific Smile and connected devices, the following platforms will be available:
+
+- [Climate](#climate) for Adam and (a stand-alone) Anna.
+- [Binary Sensor](#binary_sensor) for status of your domestic hot water or secondary heater.
```
[the remark this answers is not in the document]
Comment (CoMPaTech) [reply source/_integrations/plugwise.markdown]: Nice catch, fixed!

**Comment by CoMPaTech** [comment]
Ow, didn't comment here - but both Frenck and Joostlek have write permissions on our (plugwise-org-ed) .io repo as well to amend any small typo's.

File: source/_integrations/plugwise.markdown [hunk 2]
```diff
@@ -224,45 +194,219 @@ action: `climate.set_preset_mode`
...
+
+[Select](/integrations/select) allows for changing a thermostat schedule. If you have an Adam you can also select the regulation mode.
+
+Schedules can be created using the Plugwise App or the web-interface.
+
+#### Change climate schedule
+
+action: `select.select_option`
+
+Assuming you have something to indicate your current day of night shift based on a calendar you can automatically change the active schedule. The automation example below assumes an (`input_select`)[/integrations/input_select/] helper is present, e.g., containing something like `Day shift`, `Night shift` and `Weekend shift`). Assuming you have created the appropriate schedules using your Plugwise App, your automation for night shifts could look like:
```
[the remark this answers is not in the document]
Comment (CoMPaTech) [reply source/_integrations/plugwise.markdown]: Tnx

**Comment by CoMPaTech** [comment]
@coderabbitai full review

**Comment by CoMPaTech** [comment]
@frenck and/or @joostlek we've polished and improved to ensure even more optimal reading, looking forward to your feedback. Also see the latest comment in https://github.com/home-assistant/core/pull/133151 if we close or amend that one (referring to #https://github.com/home-assistant/home-assistant.io/pull/36087) in favour of a new Core PR when this one's up for merge.

File: source/_integrations/plugwise.markdown [hunk 3]
```diff
@@ -30,115 +30,103 @@ ha_platforms:
...
- - `select` (for changing a thermostat schedule, a regulation mode (Adam only))
- - `switch` (for Plugs connected to Adam, or Circles and Stealths connected to a Stretch)
+This integration supports one or more of the Plugwise Gateways available on your network. You can connect to these gateways using your browser, the Plugwise App, or this Home Assistant integration. There are 5 types of gateways:
 
+- Full zonecontrol using the [Adam](https://www.plugwise.com/en_US/zonecontrol) using [additional devices](#devices-overview) such as smart valves and smart-plugs.
+- A stand-alone smart thermostat called [Anna](https://www.plugwise.com/en_US/products/anna).
+- For power monitoring there is a device simply called the [P1](https://www.plugwise.com/en_US/products/smile-p1).
+- Although no longer sold, there also is support for Stretch, a gateway to create network connectivity for their older power products.
+- **Not supported yet!** The newest in the family, [Anna P1](https://www.plugwise.com/product/anna-p1/) cleverly uses energy (solar) information to transfer abundant energy in heatpump usage. [Contact us](#anna-p1) if you have one!
```
Comment (frenck) [inline source/_integrations/plugwise.markdown:43]: This is odd to list under a chapter called "Supported Devices"

As the developer documentation on IQS shows, it should be added to a chapter with unsupported devices.

File: source/_integrations/plugwise.markdown [hunk 4]
```diff
@@ -30,115 +30,103 @@ ha_platforms:
...
+
+Plugwise formerly sold power-based products using a USB stick as the controller. This integration does not support the `Stick` directly, see [legacy power devices](#legacy-power-devices) for more information.
```
Comment (frenck) [inline source/_integrations/plugwise.markdown:45]: "Stick" is not code, and should not be code fenced.

File: source/_integrations/plugwise.markdown [hunk 5]
```diff
@@ -30,115 +30,103 @@ ha_platforms:
...
+
+Depending on your specific gateway and connected devices, the following platforms will be available:
+
+- [Climate](#climate) for Adam and (a stand-alone) Anna.
+- [Binary Sensor](#binary-sensor) for status of your domestic hot water or secondary heater.
+- [Button](#button) to reboot your Plugwise Gateway.
+- [Number](#number) to change a boiler setpoint or temperature offset.
+- [Sensor](#sensor) a variety of sensors is available for all gateways and connected devices.
+- [Select](#select) to change your thermostat schedule or regulation mode.
+- [Switch](#switch) allowing plugs to be commanded.
```
Comment (frenck) [inline source/_integrations/plugwise.markdown:57]: This is not how we want to document these. For more information see: https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/docs-supported-functions

File: source/_integrations/plugwise.markdown [hunk 6]
```diff
@@ -30,115 +30,103 @@ ha_platforms:
...
 
 {% include integrations/config_flow.md %}
 
 {% configuration_basic %}
 Host:
-  description: "The hostname or IP address of your Smile. For example: `192.168.1.25`. You can find it in your router or in the Plugwise app using the **Settings** icon (&#9776;) -> **System** -> **Network**. If you are looking for a different device in the Plugwise App, on the main screen first select **Gateways** -> the Smile of your choice, and then follow the previous instruction. Normally, the Smile(s) are automatically discovered, and you don't have to provide the hostname or IP address."
+  description: "The hostname or IP address of your Gateway. For example: `192.168.1.25`. You can find it in your router or in the Plugwise app using the **Settings** icon (&#9776;) -> **System** -> **Network**. In the Plugwise App, to locate a specific device, select **Gateways** on the main screen, choose your desired gateway, and then follow the previous instructions. Normally, the Gateway(s) are automatically discovered, and you don't have to provide the hostname or IP address."
```
Comment (frenck) [inline source/_integrations/plugwise.markdown:67]: Why is "Gateway" sometimes capitalized? 

(Occurs through all of this document apparently)

File: source/_integrations/plugwise.markdown [hunk 7]
```diff
@@ -30,115 +30,103 @@ ha_platforms:
...
 Username:
-  description: "Username to log in to the Smile. This should be just `smile` - or `stretch` for a Stretch."
+  description: "Username to log in to the Gateway. This should be `smile` for most devices, or `stretch` for a Stretch."
```
Comment (frenck) [inline source/_integrations/plugwise.markdown:69]: Smile & stretch are not code and should thus not be code-fenced.

File: source/_integrations/plugwise.markdown [hunk 8]
```diff
@@ -30,115 +30,103 @@ ha_platforms:
...
+| Stretch entities |60 seconds|
 
-## Removing the integration
+## Entities
 
-This integration follows standard integration removal. No extra steps are required within Home Assistant or on your Plugwise devices.
+This integration displays all Plugwise devices in your configuration, including hardware devices, multi-thermostat climate zones, and virtual switch groups. Additionally, a device representing your Plugwise gateway (e.g., Adam, Smile-T, or P1) will be visible.
 
-{% include integrations/remove_device_service.md %}
+For example, if you have an Adam setup with a Lisa named `Living` and a Tom named `Bathroom`, these will show up as individual devices. The heating/cooling device connected to your gateway will be shown as `OpenTherm` or `OnOff`, depending on how the gateway communicates with the device. If you have Plugs (as in, pluggable switches connecting to an Adam) or Aqara Smart Plugs, those will be shown as devices as well.
```
Comment (frenck) [inline source/_integrations/plugwise.markdown:101]: Quite a few occurances of code-fencing happening here, that aren't code.

File: source/_integrations/plugwise.markdown [hunk 9]
```diff
@@ -30,115 +30,103 @@ ha_platforms:
...
-The meaning of `off` is that the Adam regulation is set to off. This means that the connected HVAC-system does not heat or cool, only the domestic hot water heating function, when available, is active.
-
-The meaning of `cool` or `heat` is that there is no schedule active. For example, if the system is manually set to cooling- or heating-mode, the system will be active if the room temperature is above/below the thermostat setpoint.
-
-The meaning of `heat/cool` is that there is no schedule active. For example, if the system is in automatic cooling- or heating-mode, the active preset or manually set temperature is used to control the HVAC system.
-
-The meaning of `auto` is that a schedule is active and the thermostat will change presets/setpoints accordingly.
+|HVAC mode|Indication|Description|
+--- | --- | ---
+|`auto` | Active schedule | The thermostat will change presets/setpoints accordingly. |
```
Comment (frenck) [inline source/_integrations/plugwise.markdown:119]: Let's avoid the use of tables with longer content, they will be impossible to read on mobile.

File: source/_integrations/plugwise.markdown [hunk 10]
```diff
@@ -156,48 +144,43 @@ action: `climate.turn_off`, `climate.turn_on` (Adam only)
...
-# Example script change the thermostat schedule
-script:
-  lisa_change_schedule:
-    sequence:
-      - action: select.select_option
-        target:
-          entity_id: select.bios_thermostat_schedule
-        data:
-          option: "Regulier"
+# Example automation using climate.turn_off for your cinema room
```
Comment (frenck) [inline source/_integrations/plugwise.markdown:150]: Please make sure this automation example follows our YAML style guide:

https://developers.home-assistant.io/docs/documenting/yaml-style-guide

File: source/_integrations/plugwise.markdown [hunk 11]
```diff
@@ -156,48 +144,43 @@ action: `climate.turn_off`, `climate.turn_on` (Adam only)
...
+  - delay:
+      seconds: 30
+mode: single
 ```
 
-#### Change boiler setpoint
+#### Update gateway data
 
-action: `number.set_value`
+Forced update of data from your gateway can be triggered by calling the generic `homeassistant.update_entity` action.
```
Comment (frenck) [inline source/_integrations/plugwise.markdown:175]: Why would we add this? In general, users shouldn't be or need to use this at all?

File: source/_integrations/plugwise.markdown [hunk 12]
```diff
@@ -224,45 +207,223 @@ action: `climate.set_preset_mode`
 
 Available options include: `home`, `vacation` (Anna only), `no_frost`, `asleep` & `away`.
 
-Example:
+Automation example:
+
+```yaml
+# Example automation changing the active (or currently set by schedule) preset
```
Comment (frenck) [inline source/_integrations/plugwise.markdown:213]: Please make sure this matched our YAML style guides.

Also move the comment/introduction comment text out. It is better to introduce it in normal text instead.

**Review by frenck — CHANGES_REQUESTED** [review]
Sorry @CoMPaTech I stopped reviewing at this point. There is just too much.

Please split this out in smaller and more iterative PRs.

../Frenck

**Comment by CoMPaTech** [comment]
Will do, tnx for the starting comments!

**Comment by frenck** [comment]
> Will do, tnx for the starting comments!

No problem! Feel free to ping me, happy to help moving them forward 🚀

**Pull request closed without merging**